documentation/modules/ROOT/pages/02_basics.adoc

```diff
@@ -121,13 +121,13 @@ NOTE: If you use `tutorial-tools`, your host `quarkus-tutorial/work` contains th
 
 == Live Coding (Development mode)
 
-Stop the Quarkus application that might be running in `jvm` or `native` mode.
+Stop the Quarkus application that might be running in JVM or native mode.
 
 _Live Coding_ or _Development mode_ enables hot deployment with background compilation, which means that when you modify your Java files and/or your resource files and refresh your browser, these changes will automatically take effect.
 
 This is probably one of the best features of Quarkus: enabling a very fast and productive iterative feedback loop.
 
-Let's start the _Live Coding_ mode by invoking `mvn`. You probably won't need to stop/start Quarkus again during this tutorial:
+Let's start the _Live Coding_ mode by using the `dev` command. You probably won't need to stop/start Quarkus again during this tutorial:
```

Comment:
You're suggesting either the CLI or Maven so let's make sure we are a bit more generic.